File: Talleres RESFTUL JAVA DOTNET/Dotnet/API-DOTNET-BANK/Application/DTOs/Account/AccountDto.cs
namespace API_DOTNET_BANK.Application.DTOs.Account
{
    public class AccountDto
    {
        public int Id { get; set; }
        public string AccountNumber { get; set; } = default!;
        public int AccountType { get; set; }
        public decimal Balance { get; set; }
        public bool IsActive { get; set; }
        public int UserId { get; set; }
        public string UserFullName { get; set; } = default!;
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }

    public class CreateAccountDto
    {
        public int AccountType { get; set; }
        public decimal InitialBalance { get; set; }
        public int UserId { get; set; }
    }

    public class UpdateAccountDto
    {
        public bool IsActive { get; set; }
    }
}

File: Talleres RESFTUL JAVA DOTNET/Java/api-java-bank/src/main/java/com/allpasoft/apijavabank/application/dto/AccountDto.java
package com.allpasoft.apijavabank.application.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AccountDto {
    private Long id;
    private String accountNumber;
    private Integer accountType;
    private BigDecimal balance;
    private Boolean isActive;
    private Long userId;
    private String userFullName;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}
